lib/practice_exercises.rb

```diff
@@ -1,13 +1,46 @@
-
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: O(n^2)
+# Space Complexity: O(1)
 def remove_duplicates(list)
-  raise NotImplementedError, "Not implemented yet"
+  list.each_with_index do |value, index|
+    if index > 0 
+      if value == list[index - 1] 
+        list.slice!(index) 
```

Comment:
Slice is going to shift subsequent elements to the left one index.  Thus it as a time complexity of O(n).  Inside this loop your method as time complexity of O(n<sup>2</sup>)